_pages/business-units/18f/projects-partners/collaboration-tools.md

```diff
@@ -172,6 +172,24 @@ Here are some common barriers to using Google Calendar and some workarounds to t
 - Truly low-impact: Mural shouldn't be used to store anything confidential or authoritative. Instead, use it for ephemeral organization of information and production visualizations. Non-GSA participants are invited to use the service anonymously.
 </details>
 
+### Feedback Collection
+
+**Qualtrics** is a feedback collection tool that helps with creating surveys. These surveys can be distributed in a variety of ways including on websites like [usa.gov](https://www.usa.gov/); and used to create data dashboards. Any TTS employee may use this tool for free. Licenses may be aquired by contacting GSA's Office of Customer Experience (OCE) at customerexperience@gsa.gov. Please make sure to state your business case in your request. 
+
+- [Qualtrics handbook page]
+
+<details markdown="block">
+<summary> Compliance and records considerations</summary>
+
+- Qualtrics has a GSA ATO and FedRAMP authorized. 
+- Qualtrics is covered under SORN GSA-CIO-3. A System of Records Notice (SORN) is related to the  Privacy Act. SORNs are under the Chief Privacy Officer's jurisdiction and are posted in the Federal Register for review. More info here on [GSA.gov](https://www.gsa.gov/reference/gsa-privacy-program/systems-of-records-privacy-act)]
+- Qualtrics should be used if you're collecting feedback that is directly helping GSA; not another agency. For example: if you're working with NIH, you cannot collect data for NIH using GSA's Qualtrics account. 
+- Please use Google Forms if you're collecting feedback from stickly internal-GSA employees. 
+- Per [GSA's Survey Order](https://insite.gsa.gov/directives-library/gsa-customer-supplier-surveys-), OCE is “responsible for ensuring survey materials distributed to recipients outside of GSA and/or across all of GSA meet agency standards.” Meaning that all Qualtrics surveys need to be approved by OCE before distribution.
+- You can find more information on Qualtrics and GSA's feedback collection efforts [HERE](https://insite.gsa.gov/services-and-offices/staff-offices/office-of-customer-experience/surveys-and-feedback-collection)
```

Comment:
> It’s most important for link text to make sense without the surrounding sentences or content.

https://usability.yale.edu/web-accessibility/articles/links#link-text

```suggestion
- [More information on Qualtrics and GSA's feedback collection efforts](https://insite.gsa.gov/services-and-offices/staff-offices/office-of-customer-experience/surveys-and-feedback-collection)
```

Reply:
Sounds good. Thanks for the link. We'll add that to our own survey guides